src/test/resources/SystemHealth.feature

```diff
@@ -7,12 +7,14 @@ Background:
 
```

Comment:
why is there not authentification?
Should every user be able to use this endpoint or only admins?

Reply:
Because its not needed in my opinion.

Reply:
The health endpoint doesnt really belong the the other rest end points.